extensions/parquet/table/build.gradle

```diff
@@ -19,6 +19,8 @@ dependencies {
     api project(':engine-stringset')
 
     implementation project(':extensions-parquet-base')
+    Classpaths.inheritParquetHadoop(project)
```

Comment:
Note to other reviewers, this was added because it should have been here to begin with - previously we picked it up transitively from our other parquet projects, but there is no actual need for it as an api dependency. Instead, this project should have declared it as an implementation dep, which it now does.